benchopt/utils/conda_env_cmd.py

```diff
@@ -112,8 +112,14 @@ def create_conda_env(env_name, recreate=False, with_pytest=False, empty=False):
     try:
         _run_shell(
             f"{CONDA_CMD} env create {force} -n {env_name} -f {env_yaml.name}",
-            capture_stdout=True, raise_on_error=True
+            capture_stdout=False, raise_on_error=True
```

Comment:
I would actually add an option `-q` to be able to shut this up.
also, you need to update `l.104,136,139` to take this into account